src/ui/FormWidgets/Checkbox.ts

```diff
@@ -114,24 +117,40 @@ export class Checkbox implements IFormWidgetWithLabel, IFormWidgetSelectable {
       'aria-label': this.ariaLabel || this.label,
       'aria-hidden': true
     }).el;
-    const button = $$('button', { type: 'button', className: 'coveo-checkbox-button', 'aria-label': this.ariaLabel || this.label });
+    this.button = $$('button', {
+      type: 'button',
+      className: 'coveo-checkbox-button',
+      'aria-label': this.ariaLabel || this.label,
+      ariaPressed: this.isSelected().toString()
```

Comment:
names could be more consistent, aria-label / ariaLabel, aria-pressed / ariaPressed

Reply:
Agreed. I'll use camel case.